### PR TITLE
Bootstrap form for card form (create/edit)

### PR DESCRIPTION
**Pull request by Fukotaku**
- Adaptation form create/edit card with bootstrap
- Autocomplete form inputs card on edit mod

Sorry for my bad english, but I have a question for your symfony project.
Why declare annotated routes directly on controllers? You have a routing.yml file for this.

And I also suggest using the php function `password_hash` for password hash rather `md5` in the future for a better security.

### CODE REVIEW

**Comment by sensorario** [comment]
Thank you for your PR. I've just opened new #27 issue for passwords.

About routing, ... if route is in annotations, I can move actions without any other work to do and no configuration files to update.

**Comment by Fukotaku** [comment]
Oh I see, It's practical

**Comment by sensorario** [comment]
I am watching your PR. About "Autocomplete form inputs card on edit mod", ... I dont see the autocomplete in edit.

**Comment by sensorario** [comment]
But the others updates are great!!! Thank you very much ^_^

**Comment by Fukotaku** [comment]
Sorry, it's auto filling data card entity in input form.
I do not know how to explain it in English.

File: app/Resources/views/card/new.html.twig [hunk 1]
```diff
@@ -4,93 +4,140 @@
...
+                          {% else %}
+                              <option value="{{year}}">{{year}}</option>
+                          {% endif %}
+                      {% endfor %}
+                      </select>
+                      /
+                      <select id="appbundle_card_datetime_date_month" name="appbundle_card[datetime][date][month]">
+                      {% for month in range(1, 12) %}
+                          {% if "now"|date('m') == month %}
+                              <option selected="selected" value="{{month}}">{{month}}</option>
```
Comment (sensorario) [inline app/Resources/views/card/new.html.twig]: @Fukotaku is this?

File: app/Resources/views/card/new.html.twig [hunk 2]
```diff
@@ -4,93 +4,140 @@
...
+      <div class="form-row">
+        <div class="form-group col-md-12">
+          Type :<br>
+          {% set counter = 0 %}
+          {% for type in types %}
+            <div class="form-check form-check-inline">
+              <label class="form-check-label required">
+                {% set counter = counter + 1 %}
+                <input type="radio" class="form-check-input" id="appbundle_card_type_{{counter}}" value="{{type.name}}" name="appbundle_card[type]"
+                {% if type.name == 'task'%}checked="checked"{% endif %}> {{type.name}}
```
Comment (sensorario) [inline app/Resources/views/card/new.html.twig]: I've also fixed this auto selection of project and card type value here https://github.com/sensorario/kanbanoise/pull/29

Comment (sensorario) [reply app/Resources/views/card/new.html.twig]: Pull https://github.com/sensorario/kanbanoise/tree/Fukotaku-master. I've fixed conflicts

File: app/Resources/views/card/new.html.twig (hunk 1 above)
Comment (Fukotaku) [reply app/Resources/views/card/new.html.twig]: Oups, it's for edit, not new. Sorry ^^'

Comment (Fukotaku) [reply app/Resources/views/card/new.html.twig]: I just add "/" for separation yyyy/mm/dd

File: app/Resources/views/card/new.html.twig (hunk 2 above)
Comment (Fukotaku) [reply app/Resources/views/card/new.html.twig]: Oups, that attribut value it's for edit, not new. Sorry ^^'

**Pull request merged**